rpcs3/Emu/RSX/VK/VKTextureCache.h

```diff
@@ -309,7 +309,7 @@ namespace vk
 			AUDIT( valid_length > 0 );
 
 			void* pixels_src = dma_buffer->map(valid_offset, valid_length);
-			void* pixels_dst = get_ptr_by_offset(valid_offset, true);
+			void* pixels_dst = get_ptr(get_section_base());
```

Comment:
I believe this is incorrect.
`get_section_base()` returns the section base (`cpu_range.start`), but we want `cpu_range.start + valid_offset`.

Should be `get_ptr(get_section_base() + valid_offset)`, I think